tests/python/gaia-ui-tests/gaiatest/visual_components/visual_component.py

```diff
@@ -0,0 +1,7 @@
+from gaiatest.apps.base import PageRegion
+
+
+class VisualComponent(PageRegion):
```

Comment:
That's also a good point. At the moment, let's focus on the switches, the checkboxes, the text input, which are all FormControl. I'll use this class name, then